web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -369,6 +370,29 @@ const EmbeddedClusterManagement = ({
   }, [nodesData?.nodes?.toString(), hasNodesChanged]);
   // #endregion
 
+  // temporary redirect logic - when user has clicked continue during initial install and went back to the cluster management page
+  useEffect(() => {
+    const appNeedsConfiguration = app?.downstream?.pendingVersions?.length > 0;
+
+    if (appNeedsConfiguration) {
+      const downstream = app.downstream;
+      const firstVersion = downstream.pendingVersions.find(
+        (version: Version) => version?.sequence === 0
+      );
+      if (
+        firstVersion?.status === "pending_preflight" ||
+        firstVersion?.status === "pending"
```

Comment:
shouldn't this redirect to the dashboard if the status is `pending`?

Reply:
@sgalsaleh Once preflight checks are done, before clicking deploy. the status is "pending".
![Screenshot 2024-08-01 at 12 14 28 PM](https://github.com/user-attachments/assets/f17c250f-1748-4818-a8ac-e0e75f798ad8)
Redirecting to dashboard doesn't seem right the user hasn't kicked off deployment.